Makefile

```diff
@@ -22,7 +22,7 @@ export LOGGING_VERSION?=5.6
 export NAMESPACE?=openshift-logging
 
 IMAGE_LOGGING_FLUENTD?=quay.io/openshift-logging/fluentd:1.14.6
-IMAGE_LOGGING_VECTOR?=quay.io/openshift-logging/vector:0.21-rh
+IMAGE_LOGGING_VECTOR?=quay.io/openshift-logging/vector:0.23-rh
```

Comment:
this change (among other vector version related changes) also coming in @vparfonov PR https://github.com/openshift/cluster-logging-operator/pull/1605

Reply:
that's alright, we can merge this one afterwards and drop this ^^^ change